packages/mobile/src/screens/form-screen/FormScreen.tsx

```diff
@@ -0,0 +1,60 @@
+import { useCallback, type ReactNode } from 'react'
+
+import { useSafeAreaInsets } from 'react-native-safe-area-context'
+
+import { Button, Flex } from '@audius/harmony-native'
+import type { ScreenProps } from 'app/components/core'
+import { Screen } from 'app/components/core'
+import { useNavigation } from 'app/hooks/useNavigation'
+
+const messages = {
+  done: 'Done',
+  clear: 'Clear'
+}
+
+export type FormScreenProps = ScreenProps & {
+  bottomSection?: ReactNode
+  onSubmit?: () => void
+  onClear?: () => void
+}
+
+export const FormScreen = (props: FormScreenProps) => {
+  const { children, style, bottomSection, onClear, onSubmit, ...other } = props
+  const navigation = useNavigation()
+  const insets = useSafeAreaInsets()
+
+  const handleSubmit = useCallback(() => {
+    navigation.goBack()
+    onSubmit?.()
+  }, [navigation, onSubmit])
+
+  return (
+    <Screen
+      variant='secondary'
+      style={[{ justifyContent: 'space-between' }, style]}
+      {...other}
+    >
+      {children}
+      <Flex
+        p='l'
+        pb={insets.bottom}
+        backgroundColor='white'
+        borderTop='default'
+        gap='m'
+      >
+        {bottomSection ?? (
+          <>
+            <Button variant='primary' fullWidth onPress={handleSubmit}>
+              {messages.done}
+            </Button>
+            {onClear ? (
+              <Button variant='secondary' fullWidth onPress={onClear}>
+                {messages.clear}
+              </Button>
```

Comment:
i think the designs have this as a plain button?

Reply:
excellent catch! will review again, i think there was an issue with the plain-button sizes and styles that werent working as expected in the design. lets pair on this later today actually and ill show you what i mean.